torch/csrc/jit/codegen/cuda/scheduler/transpose.cpp

```diff
@@ -572,7 +572,7 @@ void scheduleTranspose(Fusion* fusion, TransposeParams params) {
   //    t0
   //    |
   //   cache
-  //   /  \
+  //   |  |
```

Comment:
Does this trigger something?

Reply:
Yep it did. upstream linter sees `\` at the end and thought we are doing multiple line comment.... :rofl:

Reply:
I hate linter 😤

Reply:
wow